.github/workflows/service-ci.yml

```
@@ -178,12 +194,27 @@ jobs:
           env: development
           github-token-terraform: ${{ secrets.GH_TOKEN_TERRAFORM }}
           service-name: ${{ inputs.service-name }}
+      - name: Update Deployment Status - Success
+        if: success()
+        uses: ./.github/actions/github-deployment-status
+        with:
+          deployment-id: ${{ needs.push.outputs.deployment-id }}
+          state: success
+          description: Terraform deployment completed successfully
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)